packages/ui/v2/core/form/Select.tsx

```diff
@@ -28,6 +28,11 @@ function Select<
 >({ className, ...props }: SelectProps<Option, IsMulti, Group>) {
   return (
     <ReactSelect
+      className={classNames(
+        "cal-react-select-container block h-[36px] w-full min-w-0 flex-1 rounded-md text-sm ",
+        className
+      )}
+      classNamePrefix="cal-react-select"
```

Comment:
We add **className prefixes** `cal-react-select-container` and `cal-react-select` to allow us to target them and "apply" styling classes using tailwind